Shipfinity.Services/Implementations/SellerService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Shipfinity.DataAccess.Repositories.Interfaces;
+using Shipfinity.Services.Helpers;
+using Shipfinity.Services.Interfaces;
+using Shipfinity.Shared.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shipfinity.Services.Implementations
+{
+    public class SellerService : ISellerService
+    {
+        private readonly ISellerRepository _sellerRepository;
+        public SellerService(ISellerRepository sellerRepository)
+        {
+            _sellerRepository = sellerRepository;
+        }
+
+        public async Task ResetPasswordAsync(int sellerId, string oldPassword, string newPassword, string confirmNewPassword)
+        {
+            if (newPassword != confirmNewPassword)
+                throw new Exception("New password and confirmation do not match.");
```

Comment:
Dont throw a generic exception here, create a new BadRequestException in the shared or domain project and throw and catch that instead, catch a generic exception at the end and return a 500 internal server error